style.css

```diff
@@ -88,21 +75,332 @@ h3 {
+  padding-left: 7px;
+}
+
+.page p {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #00040a;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding: 10px;
+}
+
+.pagelinks {
+  display: flex;
+  width: 90%;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 6px;
+  padding: 15px;
+}
+
+.pagelinks li {
+  display: flex;
+  justify-content: space-around;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 10px;
+  color: black;
+  border: #888b90 solid 0.1px;
+  width: fit-content;
+  padding: 8px;
+}
+
+.button {
+  background-color: #f87213;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  font-style: normal;
+  line-height: 24px;
+  color: rgb(255, 255, 254);
+  cursor: pointer;
+  border: none;
+  width: fit-content;
+}
+
+.button:hover {
+  background-color: #726868;
+  border: none;
+}
+
+.project {
+  background-image: url(Images/Background.png);
+  background-position: center;
+  background-size: no-repeat;
+  margin: 25px;
+}
+
+.contenthead {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 44px;
+  color: #fdf6f6;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-top: 25%;
+  padding-left: 15px;
+}
+
+.contentp {
+  padding-left: 15px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f5f4f4;
+  order: 1;
+  flex-grow: 0;
+}
+
+.projectlinks {
+  display: flex;
+  width: 85%;
+  justify-content: space-around;
+  padding: 10px;
+}
+
+.projectlinks li {
+  display: flex;
+  justify-content: space-around;
+  padding: 9px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 20px;
+  color: rgb(245, 240, 240);
+  border: 0.5px none white;
+  list-style-type: none;
+  width: 30%;
+  background: #58595c;
+  opacity: 0.6;
+}
+
+.btn {
+  width: 100%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  padding: 13px;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: hsl(24, 29%, 97%);
+  background-color: #f65803;
+  border: none;
+}
+
+.aboutsection {
+  background-color: black;
+  padding-bottom: 65%;
+  background-image: url(Images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: right 270px;
+}
```

Comment:
- [ ] According to the requirements, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). So, kindly change the background-color to `#1C1A19;` instead of black color to match The Figma design 💯👌

- [ ] Kindly, apply this `background-color: #1C1A19;`  in header, about, and skills sections instead of `black` 💯 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/192225421-02933190-e962-4b9a-9aaa-9b792109a56a.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/192225002-3af73eff-5acc-4af9-92bf-581a19791886.png" width="500" /> |